media/js/firefox/australis/growth-firstrun-test1.js

```diff
@@ -0,0 +1,258 @@
+                        $('#fxa-value-prop-verified').fadeIn('fast');
+                    });
+
+                    // wipe out iframe and reposition delayed CTAs
+                    $fxaIframe.slideUp('normal', function() {
+                        $fxaIframe.remove();
+                    });
+
+                    $fxaDelayedCTAs.addClass('visible noiframe');
+                }
+            }, function(err) {
+                // on error, get rid of overlay and fall back to default content
+                $fxaOverlay.remove();
+                toggleDefaultContent(true);
+            });
+        };
+
+        // track Sync CTA click and link to about:accounts where posiible
+        trackSyncClick = function(e) {
+            e.preventDefault();
+
+            var goToAccounts = function () {
+                // available on Firefox 31 and greater
+                Mozilla.UITour.showFirefoxAccounts();
+            };
+
+            gaTrack(['_trackEvent', pageId + ' Page Interactions - New Firefox Tour', 'button click', 'Get Started with Sync'], goToAccounts);
+        };
+
+        // listen for postMessage from iframes
+        window.addEventListener('message', function(event) {
+            // listen for fxa communiques
+            if (typeof event.data === 'string') {
+                // 'ping' sent when lightbox created
+                if (event.data.indexOf('ping') > -1) {
+                    // initialize reference to fxa overlay & iframe
+                    $fxaOverlay = $('#fxa-background');
+                    $fxaIframe = $('#fxa');
+
+                    // move content into FxA overlay
+                    $fxaValueProp = $('#fxa-value-prop').detach();
+                    $fxaDelayedCTAs = $('#fxa-delayed-ctas').detach();
+
+                    $fxaOverlay.prepend($fxaValueProp).append($fxaDelayedCTAs);
+
+                    showFxA();
+                }
+                // oauth_complete sent after user verifies account
+                else if (event.data.indexOf('oauth_complete') > -1) {
+                    window.gaTrack(['_trackEvent', '/firstrun/ Optimization f35', 'Sign Up for Firefox Account', 'Verified Email Address']);
+                }
+            } else if (typeof event.data === 'object') {
+                if (event.data.addon) {
+                    window.gaTrack(['_trackEvent', '/firstrun/ Optimization f35', 'Add to Firefox', event.data.addon]);
+                }
+            }
+        });
+
+        // if chosen to show doorhanger, get tour setup
+        if (showDoorhanger) {
+            // Query if the UITour API is working before we start the tour
+            Mozilla.UITour.getConfiguration('sync', function() {
+                // hide default content
+                toggleDefaultContent(false);
+
+                // wire up sync click in default content
+                $('.sync-cta').on('click', '.button', trackSyncClick);
```

Comment:
Oh crap, that's right. That wasn't part of the intended UX - it's just built in to the iframe. Rats. Will update.